File: src/main/java/com/catan/catanbackend/model/dto/LogInForm.java
package com.catan.catanbackend.model.dto;

import lombok.AllArgsConstructor;
import lombok.Data;
import lombok.NoArgsConstructor;

@Data
@NoArgsConstructor
@AllArgsConstructor
public class LogInForm {
    private String username;
    private String password;
}

File: Assets/Scripts/User/LoginForm.cs
using System;

namespace Assets.Scripts.User
{
    [Serializable]
    public class LoginForm
    {
        public LoginForm(string username, string password)
        {
            this.username = username;
            this.password = password;
        }

        public string username;
        public string password;
    }
}
